pkg/util/util.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"strings"
+)
+
+// IntArrayToJSONString converts int array to json string
+func IntArrayToJSONString(intArray []int) string {
+	return strings.Join(strings.Split(fmt.Sprint(intArray), " "), ",")
```

Comment:
The logic is join a string array with `,`, the output is a common separated string instead of a JSON

Reply:
`fmt.Sprint([]int{1, 2, 3})` returns `[1 2 3]`. So this function is indeed converts int array to json string.

Reply:
Can we use GO JSON module to convert the GO ARRAY to JSON?
In the current implementation, we are operating on the string. I think it's not so common and readable.